src/js/components/reviewData/ReviewDataNotifyModal.jsx

```diff
@@ -107,6 +111,23 @@ export default class ReviewDataNotifyModal extends React.Component {
             });
     }
 
+    openClientEmail(e) {
+        e.preventDefault();
+        const toEmails = (this.state.selectedUsers.map((user) => user.email)).join(', ');
+        const subject = 'DATA Act Broker - Submission Ready for Review';
+        const revUser = this.props.fromUser.toUpperCase();
+        const revAgecny = this.props.submittingAgency.toUpperCase();
+        const revLink = window.location.href;
```

Comment:
We can use `react-router-dom` `location.pathname` + the base url instead of directly accessing `window.location.href`, e.g. https://reacttraining.com/react-router/web/api/Hooks/uselocation